plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js

```diff
@@ -337,9 +361,17 @@ function generateTaskDiv(taskNode) {
     $('#' + taskDivName + " #taskOutputMappingNo").val(0);
     $('#' + taskDivName + " #taskName").val(taskName.trim().replace(/ /g, ''));
     $('#' + taskDivName + ' #taskName').change(function() {
+        taskNameInput = $('#' + taskNode.getAttribute("name") + "wrapper #taskName").val();
+        if(taskNameInput.trim()!=""){
         makeDirty();
-        taskNode.setAttribute("name", $('#' + taskNode.getAttribute("name") + "wrapper #taskName").val());
+        ExecuteCustomFunction("removewsdl", taskNode.getAttribute("name"));
+        taskNode.setAttribute("name", taskNameInput);
```

Comment:
These strings were replaced by constants in https://github.com/wso2/devstudio-tooling-bps/commit/1b5789567f9361a4df08ca65bf24fd1b1f132751